src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.Color;
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final int FIGURES = 5;
+    private static final int MAX_SIZE = 99;
+    private static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
+    private static final int DEFAULT_RADIUS = 10;
+
+    public Figure getRandomFigure() {
+        return getFigure();
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
+    }
+
+    private static int randomSize() {
+        return RANDOM.nextInt(MAX_SIZE) + 1;
+    }
+
+    private Figure getFigure() {
+        ColorSuplier colorSuplier = new ColorSuplier();
+        int numberFigure = RANDOM.nextInt(FIGURES);
+        String color = colorSuplier.getRandomColor();
+        switch (numberFigure) {
+            case 1:
+                return getRandomCircle(color);
+
+            case 2:
+                return getRandomSquare(color);
+
+            case 3:
+                return getRandomRightTriangle(color);
+
+            case 4:
+                return getRandomIsoscelesTrapezoid(color);
+
+            default:
+                return getDefaultFigure();
```

Comment:
```suggestion
            case 4:
            default:
                return getRandomIsoscelesTrapezoid(color);
```

Reply:
returning default figure here isn't a good idea, it should never happen in the first place

Reply:
\+